CONTRIBUTING.md

```diff
@@ -1,43 +1,91 @@
 # Foundation Contributing Guidelines
 
-Although Foundation is maintained by ZURB, it’s also a community effort. Whether it’s bug fixing, feature development, or contributions to the ecosystem, designers and developers from all over the world help make Foundation the most advanced framework in the world.
-
-A select group of our contributors have been dubbed *Yetinauts*. They have direct write access to the codebase and support the core Foundation team on the development of the framework. Are you interested in making your mark on the Foundation framework? Whether you’re just submitting bugs or helping us write new features, there are many ways to contribute to Foundation.
-
-## Terms
-
-- The **community** is anyone commenting on issues or opening pull requests. That includes you!
-- A **Yetinaut** is anyone with write access to the repository.
-- The **Core Team** is anyone on the Foundation Team.
+Although Foundation was built and maintained by ZURB for years, it is today a community project. Whether it’s bug fixing, feature development, or contributions to the ecosystem, designers and developers from all over the world help make Foundation the most advanced framework in the world.
 
 ## Issues
 
-Open an issue for any problem you have with the framework. If there's anything missing from your issue, such as extra context, a code sample, etc. a team member will ask for more info in the comments.
-
-Support requests are generally better suited for the [Foundation Forum](http://foundation.zurb.com/forum), while GitHub is more appropriate for bugs. If you aren’t sure if your issue is a bug or not, don’t worry! Post your problem on GitHub and the team will help you along. Every participant is expected to follow the project's [Code of Conduct](code-of-conduct.md) so please be courteous and respectful.
+For **bugs**, **features requests**, **incorrect documentation** or any problem you have with Foundation, you can open a new issue. If you need help to use Foundation, the [Foundation Forum](http://foundation.zurb.com/forum) would be more appropriate. If you aren’t sure if your issue is a bug or not, don’t worry! Post your problem on GitHub and the team will help you along.
```

Comment:
features requests => feature requests  
http://foundation.zurb.com/forum => https://foundation.zurb.com/forum

Reply:
Hm, should it be `'` instead of `’`?

Reply:
Wow good catch